internal/provider/resource_docker_service_structures.go

```diff
@@ -1083,17 +1083,14 @@ func createServiceNetworks(v interface{}) ([]swarm.NetworkAttachmentConfig, erro
 func createLogDriver(v interface{}) (*swarm.Driver, error) {
 	logDriver := swarm.Driver{}
 	if len(v.([]interface{})) > 0 {
-		for _, rawLogging := range v.([]interface{}) {
```

Comment:
Why did you remove the loop?

Reply:
If you have a look at the TODO(s) there and the logic, I _think_ it only goes thorugh the loop once anyway, so the loop is superfluous. Linter also says so 😅

Reply:
Can you link me to where you found this `MAX_ELEMENTS: 1`? I'm getting a bit lost here in the code, api docs and what not 😅

Reply:
https://github.com/kreuzwerker/terraform-provider-docker/blob/master/internal/provider/resource_docker_service.go#L687